### ISSUE TITLE
Annotation coordinates mismatch on landscape-oriented pages 

### ISSUE THREAD

**Issue by joesmith0** [jsvine/pdfplumber#403]
## Describe the bug

(In the context of PDF documents with a mix of landscape and portrait-oriented pages).
On the landscape-oriented pages, the bounding boxes for highlight annotations (`x0, x1, top, bottom`) do not match the expected values for the words they should encapsulate. `x0` can be negative in these cases which suggests the annotation thinks the page is in a portrait orientation.

## Code to reproduce the problem
```
with pdfplumber.open(fn) as pdf:
    annot = pdf.annots[0]
    print({key: annot[key] for key in annot if key in ['x0', 'x1', 'top', 'bottom']}, end='\n')
    
    page = pdf.pages[annot['page_number']-1]
    words = [word for word in page.extract_words() if word['text'] in ['December', '2019']]
    
    for word in words:
        print(
            str({key: word[key] for key in word if key in ['x0', 'x1', 'top', 'bottom', 'text']})
            + '\n'
        )
```

## Expected behavior
The bounding boxes of the words outputted above should be contained within the bounding box of the annotation.

## Actual behavior
The bounding box of the annotation is completely off the target values (sometimes negative).

## Environment

- pdfplumber version: 0.5.26
- Python version: 3.6.12
- OS: Mac

**Comment by samkit-jain** [comment]
Hi @joesmith0 Appreciate you using the library and raising a bug report. In order to further investigate this, request you to share a PDF that demonstrates this issue as well. Please remove any sensitive information from the PDF before sharing it.

**Comment by SasiAravind** [comment]
Hi @samkit-jain @jsvine   Is there any way to search keywords containing space e.x =['Team size','Company Name'].
while using **"page.extract_words()"**

**Comment by joesmith0** [comment]
Hey @samkit-jain, unfortunately I wasn't able to get the redaction tool to work on my PDFs... Do you have any PDFs at hand with a mix of portrait and landscape pages? The characters should all be upright (same direction) despite the orientation.

**Comment by jsvine** [comment]
Closing this issue due to inactivity, lack of issue-reproducing PDF, and lack of other users expressing similar issues. Feel free to continue the discussion, however, especially if someone comes across a PDF that allows us to reproduce.

**Comment by jsvine** [comment]
Just leaving a note here that the likely issue (hard to confirm given lack of PDF) is likely fixed in [v0.11.1](https://github.com/jsvine/pdfplumber/releases/tag/v0.11.1) — specifically commit [aaa35c9](https://github.com/jsvine/pdfplumber/commit/aaa35c9).
